app/src/main/java/org/lfenergy/compas/scl/data/rest/CompasDataResource.java

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: 2021 Alliander N.V.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.lfenergy.compas.scl.data.rest;
+
+import org.lfenergy.compas.scl.data.model.SclType;
+import org.lfenergy.compas.scl.data.rest.model.*;
+import org.lfenergy.compas.scl.data.service.CompasDataService;
+
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.util.UUID;
+
+@Path("/scl/v1")
```

Comment:
Good to document somewhere how we do REST versioning.
In the URL is one of the options (also my preference), but we need to document it somewhere :)

Also, should this be configurable instead of hardcoded?